usage/usageplot.py

```diff
@@ -29,11 +25,16 @@ def create_parser():
                         'if omitted, there will be no title on the plot.')
     p.add_argument('-d', '--data', type=str, default=None,
                    help='Name of the usage output file made from usage.py, '
-                        'if omitted, usage.out in the current directory will be used.')
+                        'if omitted, usage.out in the current directory will '
+                        'be used.')
```

Comment:
I'm not sure if making file with data to plot an optional argument (with some arbitrary default value) is a good idea. It seems to violate [the principle of least astonishement](https://en.wikipedia.org/wiki/Principle_of_least_astonishment). When I'm trying to plot something I expect I'll have to provide the source of the data. After all, I want to see *my* data, not a content of some (arbitrarily named) file.